charts/astronomer/templates/houston/api/houston-deployment.yaml

```diff
@@ -114,11 +114,15 @@ spec:
             initialDelaySeconds: {{ .Values.houston.readinessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.houston.readinessProbe.periodSeconds }}
             failureThreshold: {{ .Values.houston.readinessProbe.failureThreshold }}
+          {{- if .Values.houston.command }}
+          command:
+            {{- toYaml .Values.houston.command | nindent 12 }}
+          {{ else }}
+          command: ["yarn", "serve"]
+          {{- end }}
```

Comment:
@mishah334 - if no command block is mention it will take from defaults , should we still need to override them